components/integrations/installed_outgoing_webhooks/installed_outgoing_webhooks.test.jsx

```diff
@@ -53,8 +55,7 @@ describe('components/integrations/InstalledOutgoingWebhooks', () => {
 
     test('should match snapshot', () => {
         function emptyFunction() {} //eslint-disable-line no-empty-function
-
-        const wrapper = shallow(
+        const wrapper = mountWithIntl(<BrowserRouter>
```

Comment:
maybe instead of doing a mount of the entire component we can make a shallow, and then create a snapshot of the execution of the `children` property passing the `filter` state.

Reply:
Great idea. Will do